packages/triple-document/src/elements/shared/display-containers.tsx

```diff
@@ -0,0 +1,50 @@
+import styled from 'styled-components'
+import { Fragment, ReactNode } from 'react'
+import { ImageMeta } from '@titicaca/type-definitions'
+import {
+  Container,
+  ImageBlockElementContainer,
+  ImageCarouselElementContainer,
+} from '@titicaca/core-elements'
+
+import DocumentCarousel from './document-carousel'
+
+function BlockContainer({
+  children,
+  images,
+}: {
+  children: ReactNode
+  images: ImageMeta[]
+}) {
+  return (
+    <Container
+      margin={{
+        top: 40,
+        bottom: images.some(({ title }) => title) ? 10 : 30,
+      }}
+    >
+      {children}
+    </Container>
+  )
+}
+
+const GridContainer = styled.div`
+  display: grid;
+  grid-template-columns: repeat(auto-fit, minmax(0, auto));
+`
+
+export const imagesContainerMap = {
```

Comment:
파일스코프의 상수니까 ALL_CAPS로 네이밍하는 게 규칙이었던 것 같아요!

```suggestion
export const IMAGES_CONTAINER_MAP = {
```

Reply:
`ALL_CAPS`로 수정하겠습니다! 🙇🏾‍♂️